packages/api-request-builder/src/query-projection.js

```diff
@@ -6,7 +6,7 @@
  * (for current / published)
  * @return {Object} The instance of the service, can be chained.
  */
-export function staged(isStaged: boolean = true): Object {
+export function staged(isStaged: boolean = false): Object {
```

Comment:
While we're here. Maybe `shouldFetchStaged` might be a better name? _Note_: of the parameters not the `param`.